htdocs/adherents/subscription.php

```diff
@@ -927,7 +927,7 @@
 
                 // Bank account
                 print '<tr class="bankswitchclass"><td class="fieldrequired">'.$langs->trans("FinancialAccount").'</td><td>';
-                $form->select_comptes(GETPOST('accountid'), 'accountid', 0, '', 1);
+                $form->select_comptes(GETPOST('accountid'), 'accountid');
```

Comment:
If there is several bank account, it will automatically set the bank to the first one, leading to a lot of errors.
A better fix will be to use the value 2 instead of 1 for parameter useempty (2 means, add empty value only there is 2 or more 1 bank accounts)
$form->select_comptes(GETPOST('accountid'), 'accountid', 0, '', 2);